openml/extensions/sklearn/extension.py

```diff
@@ -1531,15 +1534,31 @@ def _prediction_to_probabilities(y: np.ndarray, classes: List[Any]) -> np.ndarra
             -------
             np.ndarray
             """
+
+            if isinstance(task, (OpenMLClassificationTask, OpenMLLearningCurveTask)):
+                print("\n\nInside _prediction_to_probabilities\n\n")
+                if task.class_labels is not None:
+                    if isinstance(y_train, np.ndarray) and isinstance(task.class_labels[0], str):
+                        # mapping (decoding) the predictions to the categories
+                        # creating a separate copy to not change the expected pred_y type
+                        preds = [task.class_labels[pred] for pred in y]
+                        y = preds
+                else:
+                    raise ValueError("The task has no class labels")
+                classes = model_classes
+            else:
+                return None
+
             # y: list or numpy array of predictions
             # model_classes: sklearn classifier mapping from original array id to
             # prediction index id
             if not isinstance(classes, list):
                 raise ValueError("please convert model classes to list prior to " "calling this fn")
```

Comment:
Looks like the initial black refactor de-wrapped this line wrongly (please remove the redundant space and quotes).